### ISSUE TITLE
s2i build fails because package is not found

### ISSUE THREAD

**Issue by pacospace** [thoth-station/s2i-thoth#201]
**Describe the bug**
```
2021-07-23 12:37:17,704 [11] DEBUG    thamos.config: Writing configuration file to '/tmp/src/.thoth.yaml'
>>> Thoth's configuration file after hardware and software discovery:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'yaml'
```

**Expected behavior**

**Actual behavior**

**Additional context**

**Build log**

**Comment by fridex** [comment]
It looks like Thoth tooling packages are not installed in an appropriate location. They should be installed into system packages, not to collide with user packages brought during the container image build process. ubi8-py39 has this correctly set. Might be worth switching to the same Dockerfile structure as for other container images as well.

**Comment by fridex** [comment]
> It looks like Thoth tooling packages are not installed in an appropriate location. They should be installed into system packages, not to collide with user packages brought during the container image build process. ubi8-py39 has this correctly set. Might be worth switching to the same Dockerfile structure as for other container images as well.

But it looks like ubi8-py39 does not correctly set Thoth assemble addition.

**Comment by pacospace** [comment]
> It looks like Thoth tooling packages are not installed in an appropriate location. They should be installed into system packages, not to collide with user packages brought during the container image build process. ubi8-py39 has this correctly set. Might be worth switching to the same Dockerfile structure as for other container images as well.

Solved by: https://github.com/thoth-station/s2i-thoth/pull/209

**Comment by pacospace** [comment]
The issue is no more there, I tested the s2i-example scenario with changes and correct image: https://quay.io/repository/pacospace/ubi8-py38?tab=tags

**Comment by pacospace** [comment]
> > It looks like Thoth tooling packages are not installed in an appropriate location. They should be installed into system packages, not to collide with user packages brought during the container image build process. ubi8-py39 has this correctly set. Might be worth switching to the same Dockerfile structure as for other container images as well.
> 
> But it looks like ubi8-py39 does not correctly set Thoth assemble addition.

what is the issue? the image builds correctly for me.

**Comment by pacospace** [comment]
> > It looks like Thoth tooling packages are not installed in an appropriate location. They should be installed into system packages, not to collide with user packages brought during the container image build process. ubi8-py39 has this correctly set. Might be worth switching to the same Dockerfile structure as for other container images as well.
> 
> Solved by: #209

```
(app-root) pip list
Package    Version
---------- -------
pip        20.3.3
setuptools 41.6.0

```
